### PR TITLE
MAG-289 [ORFS-97] Add create delete raid graph

### PR DESCRIPTION
**Pull request by WangWinson**
This PR is to add graph for create or delete raid then refresh the MegaRAID catalog
Dependency PR https://github.com/RackHD/on-tasks/pull/76

@RackHD/corecommitters @iceiilin @pengz1 @sunnyqianzhang

### CODE REVIEW

File: spec/lib/graphs/delete-megaraid-graph-spec.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+describe(require('path').basename(__filename), function () {
+    var base = require('./base-graph-spec');
+
+    base.before(function (context) {
+        context.taskdefinition = helper.require('/lib/graphs/delete-megaraid-graph.js');
```
Comment (WangWinson) [inline spec/lib/graphs/delete-megaraid-graph-spec.js]: Hound doesn't mark global.helper as a lint exception.

File: lib/graphs/create-megaraid-graph.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        },
+        {
+            label: 'refresh-catalog-megaraid',
+            taskName: 'Task.Catalog.megaraid',
+            waitOn: {
+                'create-raid': 'succeeded'
+            }
+        }
+    ]
+};
```
Comment (yyscamper) [inline lib/graphs/create-megaraid-graph.js]: Usually this kind of workflow should be end with a "reset" task, we don't want the node stays in micro-kernel.

File: lib/graphs/delete-megaraid-graph.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        },
+        {
+            label: 'refresh-catalog-megaraid',
+            taskName: 'Task.Catalog.megaraid',
+            waitOn: {
+                'delete-raid': 'succeeded'
+            }
+        }
+    ]
+};
```
Comment (yyscamper) [inline lib/graphs/delete-megaraid-graph.js]: Usually this kind of workflow should be end with a "reset" task, we don't want the node stays in micro-kernel.

Comment (WangWinson) [inline lib/graphs/delete-megaraid-graph.js]: Good point ! Will update

File: spec/lib/graphs/delete-megaraid-graph-spec.js (hunk 1 above)
Comment (benbp) [inline spec/lib/graphs/delete-megaraid-graph-spec.js]: Figured it out, hound is not reading our test .jshintrc in the spec directory https://github.com/thoughtbot/hound/issues/889

**Comment by benbp** [comment]
:+1:

**Comment by yyscamper** [comment]
:+1:

File: spec/lib/graphs/delete-megaraid-graph-spec.js (hunk 1 above)
Comment (WangWinson) [inline spec/lib/graphs/delete-megaraid-graph-spec.js]: Thanks @benbp. Good to know this kind of info

**Pull request merged**